schulcloud/DashboardViewController.swift

```diff
@@ -7,14 +7,18 @@
 //
 
 import UIKit
-import CoreData
-
 
 protocol ViewControllerHeightDataSource: class {
     var height : CGFloat { get }
 }
 typealias DynamicHeightViewController = UIViewController & ViewControllerHeightDataSource
 
+extension User {
+    var canDisplayNotification : Bool {
```

Comment:
Do we need this extension? We don't do the same for `. homeworkView`.

Reply:
It's a small readability gain because we check more often for notification permissions, homework is checked once on init.